charts/fluid/fluid/templates/role/dataset/rbac.yaml

```diff
@@ -13,6 +13,14 @@ rules:
     - get
     - list
     - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
```

Comment:
Can we only use rbac `get`?

Reply:
Please refer https://github.com/fluid-cloudnative/fluid/pull/1526. Thanks.

Reply:
The commit use `apiReader` is under test, will be pushed soon.

Reply:
fixed.